Snakemake/workflow/rules/merge_samples_vcf.smk

```diff
@@ -4,8 +4,8 @@ import yaml
 if len(allFiles) != 1:
     rule get_samples:
         input:
-            f'{vcfdir}/{{chromosome}}/{{chromosome}}_{{file}}.vcf.gz'
-        output: temp(f'{vcfdir}/{{chromosome}}/{{chromosome}}_{{file}}.txt')
+            f'../{Project}/VCF/{{chromosome}}/{{chromosome}}_{{file}}.vcf.gz'
+        output: temp(f'../{Project}/VCF/{{chromosome}}/{{chromosome}}_{{file}}.txt')
         conda: "bcftools"
         threads: 1
```

Comment:
I think what Isidore was doing makes sense. Not sure if my suggested change here broke some of his suggested changes? Happy to look into using {workdir}.

Reply:
Re Project/VCF, I understand your point with keeping these files for later. I think however it would be good to leave the original data folder untouched by default (for instance, we may not want to change the folders with the test data I guess).

If do we want to write into the original VCF dir, we could tell Snakelike to work in the original data directory by setting  {workdir} to that location? To be implemented soon...

Reply:
OK, I have made the processing of the VCF in the original dir optional. Option added to the config file `stinger_Eddie.yaml`:
`process_vcf_in_original_dir: false # makes a VCF dir inside the output dir. Set to true to stay in original dir.`

Reply:
I think makes sense having the processed vcf files in the original folders - I need phased vcfs for any analysis of this data set and phasing is a very demanding process so makes sense doing it only once. Also, the modifications we do to the vcfs to be able to run tsinfer are sensible edits for other analysis as well. 
Moving it to this project folder is not bad neither although _for me_ makes more sense keeping them in the original place.

Reply:
Understood. Either way is now possible. @gmafrafortuna , you would set `process_vcf_in_original_dir: true`